proto/api_key.proto

```diff
@@ -16,6 +16,20 @@ message ApiKey {
   // Optional. The user-specified label of this API key that helps them
   // remember what it's for.
   string label = 3;
+
+  // A capability associated with an API key.
+  //
+  // Values are powers of 2 so that bitmask operations can be used
+  // to check capabilities.
+  enum Capability {
+    UNKNOWN_CAPABILITY = 0;
+    // Allows the API key to write to the content-addressable store
+    // and action cache.
+    CACHE_WRITE = 1; // 2^0
```

Comment:
nit: CACHE_WRITE_CAPABILITY to be consistent with enum naming style

Reply:
Done